starter-code/InvestingPortfolio/app/src/main/java/drewmahrt/generalassemb/ly/investingportfolio/MainActivity.java

```diff
@@ -57,13 +61,46 @@ protected void onCreate(Bundle savedInstanceState) {
         Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
         setSupportActionBar(toolbar);
 
-        mPortfolioRecyclerView = (RecyclerView) findViewById(R.id.portfolio_list);
+        mPortfolioListView = (RecyclerView)findViewById(R.id.portfolio_list);
+
+        mCursorAdapter = new CursorAdapter(this,null,0) {
+            @Override
+            public View newView(Context context, Cursor cursor, ViewGroup parent) {
+                return LayoutInflater.from(context).inflate(android.R.layout.simple_list_item_2,parent,false);
+            }
+
+            @Override
+            public void bindView(View view, Context context, Cursor cursor) {
+                TextView text1 = (TextView) view.findViewById(android.R.id.text1);
+                TextView text2 = (TextView) view.findViewById(android.R.id.text2);
+
+
+                String name = cursor.getString(cursor.getColumnIndex("stockname"));
+                String symbol = cursor.getString(cursor.getColumnIndex("symbol"));
+                String quantity = cursor.getString(cursor.getColumnIndex("quantity"));
 
-        mAdapter = new StockRecylerViewAdapter(new ArrayList<Stock>());
-        mPortfolioRecyclerView.setLayoutManager(new LinearLayoutManager(this,LinearLayoutManager.VERTICAL,false));
-        mPortfolioRecyclerView.setAdapter(mAdapter);
 
+                text1.setText(name+" ("+symbol+")");
+                text2.setText("Quantity of stocks: "+quantity);
+            }
+        };
+
+        mPortfolioListView.setOnLongClickListener(new AdapterView.OnLongClickListener() {
```

Comment:
You're trying to set an OnLongClickListener on the whole RecyclerView, which doesn't make any sense. You might be thinking of _OnItemLongClickListener_ which is something you can set on a ListView, but there's no such "OnItem..." listeners for RecyclerView. You have to set the listeners on each individual item in your adapter.